### ISSUE TITLE
Reported post/moderation notification missing?

### ISSUE THREAD

**Issue by Spuds** [elkarte/Elkarte#2941]
http://www.elkarte.net/community/index.php?topic=4484

**Comment by emanuele45** [comment]
Works on my localhost

**Comment by emanuele45** [comment]
Indeed, it may be worth moving this stuff to notifications (for all the flexibility involved), but I wouldn't do it now, because it has implications I think.
